.cspell.yml

```diff
@@ -0,0 +1,37 @@
+# For a description of configuration options, see
+# https://www.streetsidesoftware.com/vscode-spell-checker/docs/configuration/
+# cSpell:ignore textlint textlintrc
+version: '0.2'
+caseSensitive: true
+ignorePaths:
+  - '*.svg'
+  - node_modules
+  - public
+  - tmp
+patterns:
+  - name: CodeBlock
+    pattern: |
+      /
+          ^(\s*[~`]{3,})   # code-block start
+          .*               # all languages and options, e.g. shell {hl_lines=[12]}
+          [\s\S]*?         # content
+          \1               # code-block end - cSpell:disable-next-line
+      /igmx
+  # Temporary until we've removed the `children` entries from front matter
+  - pattern: '^url:\s+(cassandra|openshift|podman|kubernetes$'
+    description: Ignore url fields in front matter (and elsewhere)
+languageSettings:
+  - languageId: markdown
+    ignoreRegExpList:
+      - CodeBlock
+dictionaryDefinitions:
+  - name: project-words
+    path: .cspell/project-words.txt
+  - name: project-names
+    path: .cspell/project-names.g.txt
+dictionaries:
+  - project-words
+  - project-names
+words:
+  - CNCF
```

Comment:
Can we keep these in the dictionary?

Reply:
Oh, this is a convention we have in other projects where we put locale independent terms here. Let me know if you still want me to move them.

Reply:
we have project-words.txt, I don't think it makes sense to have yet another location where words are defined